src/app/components/cookie-policy/cookie-policy.component.ts

```diff
@@ -1,40 +1,137 @@
 import { Component } from '@angular/core';
+import { RpxTranslationService } from 'rpx-xui-translation';
+
+interface CookieDetails{
+  name: string;
+  cat: string;
+  purpose: string;
+  purposeWelsh: string;
+  expires: string;
+}
 
 @Component({
   selector: 'exui-cookie-policy',
-  templateUrl: './cookie-policy.component.html'
+  templateUrl: './cookie-policy.component.html',
 })
 export class CookiePolicyComponent {
-
-  public readonly googlePurpose = 'This helps us count how many people visit the service by tracking if you\'ve visited before';
-// Ideally this would be an enum but angular can't seem to cope with enums in templates
+  public readonly googlePurpose =
+    `This helps us count how many people visit the service by tracking if you've visited before`;
```

Comment:
We don't need to use those quotes unless there is string interpolation going on.

Reply:
`public readonly googlePurpose = 'This helps us count how many people visit the service by tracking if you\'ve visited before';`
Do you prefer using escape characters over back ticks? - just find it modern and easier to read if your string has a few special characters
`public readonly googlePurpose = `This helps us count how many people visit the service by tracking if you've visited before`;`